webui/react/src/components/InfoBox.tsx

```diff
@@ -5,25 +5,40 @@ import css from './InfoBox.module.scss';
 export interface InfoRow {
   content?: React.ReactNode;
   label: string;
+  onClick?: () => void,
+}
+
+export enum InfoboxStyle {
+  Default,
+  Boxed,
 }
 
 interface Props {
   rows: InfoRow[];
+  style?: InfoboxStyle,
 }
 
-export const renderRow = ({ label, content }: InfoRow): React.ReactNode => {
+export const renderRow = ({ content, label, onClick }: InfoRow): React.ReactNode => {
```

Comment:
**suggestion:** how about using `OverviewStats` component for this, since it's a similar style (just need to update it to support links).